lib/ExtUtils/MM.pm

```diff
@@ -61,7 +61,8 @@ if( $^O eq 'MSWin32' ) {
 $Is{UWIN}   = $^O =~ /^uwin(-nt)?$/;
 $Is{Cygwin} = $^O eq 'cygwin';
 $Is{NW5}    = $Config{osname} eq 'NetWare';  # intentional
-$Is{BeOS}   = ($^O =~ /beos/i or $^O eq 'haiku');
+$Is{BeOS}   = $^O =~ /beos/i;
```

Comment:
Does keeping `$Is{BeOS}` even make sense?

Reply:
I don't know, I should ask

Reply:
I suppose we should keep it for now, and re-evaluate when we look at  #290